Sources/CardPayments/Models/Vault.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+/// Used to configure Vault request
+public struct Vault {
+    
+    /// Optional. When set, the value of this property will be used to associate a payment method with the vault of a customer with this ID. If not specified a customer ID will be generated.
```

Comment:
Take it or leave it to break this out onto two lines since it's a bit long:
```suggestion
    /// Optional. When set, the value of this property will be used to associate a payment method with the vault of a customer with this ID. 
    /// If not specified a customer ID will be generated.
```